layers/src/main/scala/geotrellis/layers/hadoop/HadoopCollectionLayerProvider.scala

```diff
@@ -11,7 +11,7 @@ import java.net.URI
 
 
 class HadoopCollectionLayerProvider extends AttributeStoreProvider with ValueReaderProvider with CollectionLayerReaderProvider {
-  val schemes: Array[String] = Array("hdfs", "hdfs+file", "s3n", "s3a", "wasb", "wasbs")
+  import HadoopCollectionLayerProvider._
```

Comment:
This seems like indirection for indirection's sake. I'm not overly offended by it, but it may be worth rethinking

Reply:
My main reason for moving the `schemes` to the `HadoopCollectionLayerProvider` object is because they're needed to check the URI: https://github.com/locationtech/geotrellis/blob/e66aa5a59c0419993a220398219e7345d3245207/layers/src/main/scala/geotrellis/layers/hadoop/util/HdfsRangeReaderProvider.scala#L14

Reply:
I ended up moving the `SCHEMES` to the Hadoop `package` object.